src/test/java/no/unit/nva/handlers/authorizer/LambdaAuthorizerTest.java

```diff
@@ -0,0 +1,150 @@
+package no.unit.nva.handlers.authorizer;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.secretsmanager.AWSSecretsManager;
+import com.amazonaws.services.secretsmanager.model.GetSecretValueRequest;
+import com.amazonaws.services.secretsmanager.model.GetSecretValueResult;
+import com.amazonaws.services.secretsmanager.model.ResourceNotFoundException;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+import java.util.Map;
+import no.unit.nva.database.interfaces.WithEnvironment;
+import no.unit.nva.testutils.HandlerRequestBuilder;
+import nva.commons.handlers.authentication.AuthorizerResponse;
+import nva.commons.handlers.authentication.StatementElement;
+import nva.commons.utils.Environment;
+import nva.commons.utils.JsonUtils;
+import org.apache.http.HttpHeaders;
+import org.junit.jupiter.api.Test;
+import org.mockito.invocation.InvocationOnMock;
+import org.mockito.stubbing.Answer;
+
+public class LambdaAuthorizerTest implements WithEnvironment {
+
+    public static final String CORRECT_SECRET_NAME = "someSecretName";
+    public static final String CORRECT_SECRET_KEY = "someSecretKey";
+    public static final String CORRECT_SECRET_VALUE = "someSecretValue";
+    public static final String AWS_SERCRETS_MANAGER_ERROR_MESSAGE = "AwsSercretsManager message";
+    public static final String DEFAULT_ENV_VALUE = "*";
+    public static final int SINGLE_STATEMENT_EXPECTED = 1;
+    public static final int UNIQUE_STATEMENT = 0;
+    public static final String ACTION_ALLOWED = "Allow";
+    public static final String ACTION_DENIED = "Deny";
+
+    public static final String DEFAULT_METHOD_ARN = "arn:aws:execute-api:eu-west-1:884807050265:2lcqynkwke/Prod/GET"
+        + "/service/users/orestis@unit.no";
```

Comment:
This is a real ARN? Any chance it will actually try to connect to a real resource?